module/ptu.js

```diff
@@ -861,3 +861,33 @@ Hooks.on("preCreateItem", async function (item, data, options, sender) {
   // In preCreate[document] hook, you can update a document's data class using `document.data.update` before it is committed to the database and actually created.
   await item.data.update({ "data.origin": origin });
 });
+
+class dexButton extends Application {
+  constructor() {
+      super();
+  }
+}
+
+Hooks.on('getSceneControlButtons', function(hudButtons) {
+  let hud = hudButtons.find(val => { return val.name == "token"; })
```

Comment:
```suggestion
  const hud = hudButtons.find(val => val.name == "token")
```
Simple one-liner instead of full anon function.